### PR TITLE
Added Functionality To Play Single Track

### PR DESCRIPTION
**Pull request by astrowave**
Currently works as expected in the local collection but could do with testing on Spotify and GMusic accounts as I am unable to test that.  Should be fine.

### CODE REVIEW

File: __init__.py [hunk 1]
```diff
@@ -22,10 +22,10 @@ def __init__(self):
...
     def _connect(self, message):
         url = 'http://localhost:6680'
-        if self.settings:
-            url = self.settings.get('mopidy_url', url)
-        if self.config:
-            url = self.config.get('mopidy_url', url)
+#        if self.settings:
+#            url = self.settings.get('mopidy_url', url)
+#        if self.config:
+#            url = self.config.get('mopidy_url', url)
```
Comment (forslund) [inline __init__.py:28]: Did you intend to comment out this block to get mopidy url?

**Review by forslund — COMMENTED** [review]
Looks good, I'll give it a run through when I get back home.

Thank you for your contributions!

File: __init__.py (hunk 1 above)
Comment (astrowave) [reply __init__.py:28]: Ah, I had forgot to revisit that bug.  Yes I meant to comment it out initially as for me at least it couldn't get config and so the skill didn't connect to mopidy.

**Comment by forslund** [comment]
The config member was dropped as part of 19.08 so the lines relating to
that should be removed but I think settings should remain.

Den lör 23 nov. 2019 09:53astrowave <notifications@github.com> skrev:

**Comment by forslund** [comment]
I restored the settings and merged this. Many thanks!

**Pull request merged**